src/ndi-source.cpp

```diff
@@ -870,8 +862,8 @@ void ndi_source_thread_process_video2(ndi_source_config_t *config,
 		break;
 
 	default:
-		blog(LOG_INFO,
-		     "[obs-ndi] warning: unsupported video pixel format: %d",
+		blog(LOG_WARNING,
```

Comment:
@paulpv do you remember the argument for this upgrade to Warning?
Not sure if this was because we got some Issues report about it recently or not.

Reply:
I changed the level to `LOG_WARNING` to match that the text says "warning".

Reply:
Resolving